### DISCUSSION TITLE
Refund Chrome license or partially Firefox license

### DISCUSSION THREAD

**Discussion by wadid** [aclap-dev/video-downloadhelper#998] [License & Purchase requests]
### What's your issue?

I realized that the license that I bought is bound to Chrome and now pretty much useless for my main use case to download from Youtube.

I now just bought the Firefox license additionally. Pls refund me the Chrome license. Or alternatively refund me on the Firefox license the price of the Chrome license.

Thank you very much!

First 10 digits on Chrome license:  fc0756d0de
First 10 digits on Firefox license: e65491f153

### The 10 first digits (only 10) of your license number or email address

fc0756d0de

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
I sent you an email.

**Comment by Asleik** [comment]
Hi. I would like to to the same. This is my key for Chrome:

13f1b5bc46 (first 10 digits)

**Reply by jcfrog** [reply]
Sorry, refund is only possible one month after purchase, your Chrome license is from 2021.

**Reply by felipy361** [reply]
Olá. Gostaria de fazer o mesmo. Esta é a minha chave para o Chrome: 16d00b4e1f (primeiros 10 dígitos)

**Comment by ChaosTh3o7y** [comment]
Hi! im experiencing the same issue here. the first 10 of my chrome key is 79b456c5af

**Reply by jcfrog** [reply]
I sent you an email.

**Reply by ChaosTh3o7y** [reply]
I don’t want a refund! I’m happy to pay, I just need the license for Firefox and not chrome

**Reply by jcfrog** [reply]
We can't swap licenses. Chrome and Firefox premium are separate products with different prices and features.
What I can do is refund your Chrome premium so that you can purchase Firefox premium if you want to.
Do you want me to do this ?

**Reply by ChaosTh3o7y** [reply]
Yes that works thank you

**Reply by jcfrog** [reply]
You have been refunded. Should be available in a few business days.

**Reply by GrampaWildWilly** [reply]
But there is nothing stopping you from buying a Firefox license now.  Like Jérôme said, VDH for Chrome & VDH for Firefox are separate products.  There's nothing to stop you from buying a Firefox license now & getting your refund later.  A number of our fellow users here have both licenses so you don't need the refund before you can buy the Firefox license.  The refund will arrive whenever your bank condescends to grant you your money back.  Have you ever noticed how fast they are at taking your money but they're never that fast at giving it back?  In any case, you can buy now & get your refund later.

**Reply by delroypearson** [reply]
_We can't swap licenses. Chrome and Firefox premium are separate products with different prices and features.
What I can do is refund your Chrome premium so that you can purchase Firefox premium if you want to.
Do you want me to do this ?_  ** I would like the same as well, **

> We can't swap licenses. Chrome and Firefox premium are separate products with different prices and features. What I can do is refund your Chrome premium so that you can purchase Firefox premium if you want to. Do you want me to do this ?

**I would like same ... where do I begin?**

**Reply by jcfrog** [reply]
send us an email at licenses@downloadhelper.net with the email address you used for purchase.

**Comment by RicM88** [comment]
Oi Gostaria De Pedir reembolso Da Minha licença Por Favor
Primeiros 10 Números
d24d94598a

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by alandonadel** [comment]
Oi Gostaria De Pedir reembolso Da Minha licença Por Favor
Primeiros 10 Números

Hi, I would like to request a refund for my license, please.
First 10 Numbers
b6cf46afc6

**Reply by jcfrog** [reply]
Sorry, the refund is possible the first month, I can't refund this license from June.

**Reply by alandonadel** [reply]
It's a shame, because it was a license that was supposed to be life-time and it only lasted a few months

**Reply by jcfrog** [reply]
It is a lifetime license. You probably just need to revalidate it https://github.com/aclap-dev/video-downloadhelper/wiki/V9-interface#validate-your-license-key

**Comment by garunaj19** [comment]
Subject: Request to Transfer Chrome License to Firefox

Dear sir,

I hope this message finds you well. I am writing to request the transfer of my Chrome license to Firefox. I believe this change would better suit my current needs.

My first 10 numbers - 28e19977b9


Thank you for your consideration. I appreciate your assistance in facilitating this request.

Best regards,  
Arunaj

**Reply by jcfrog** [reply]
Hi,
Sorry we can't swap licenses.
Chrome and Firefox premium are separate products with different prices and features.

**Comment by AverySlice** [comment]
Hi. When I went to buy a license for Chrome (I use opera but opera supports chrome extensions) it automatically redirected me to the firefox license purchase screen. I would like a refund please. The first ten numbers of my license are: be41747650

Thanks,
Avery

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by marcioscavassa** [comment]


**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by ivanrobledo92** [comment]
I bought the Edge license today, but I can't download on YouTube. In Firefox there is a beta version that is possible. I would like to request a refund for my license so I can purchase it in Firefox.

The first 10 digits are: 9a1f71f1af

Thank you

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by AleOpuchkevich** [comment]
Hi, I would like to request a refund for my license, please.
First 10 Numbers
5bb1d05de3

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by Finestlotus** [comment]
Hi, I would also like to ask for a refund of my license.
I accidently bought the wrong one.
First 10 digits are: fe12fa7e04

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by ayykb** [comment]
Hello Downloadhelper i would like to request a refund for DownloadHelper Video Conversion License for Firefox . The 10 digits on firefox license : d25ae265e4 thankyou

**Comment by ronniealdrin** [comment]
Same here. I would like to receive the refund from my Chrome Licence to buy a Firefox license. I bought last week.
First 10 digits: 89da91eded
Thank you.

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by ronniealdrin** [comment]
Hi,
Thank you! :)
    Em quarta-feira, 12 de fevereiro de 2025 às 19:10:13 GMT+1, jcfrog ***@***.***> escreveu:  
 
 


Hi,
You have been refunded. Should be available in a few business days.

**Comment by Arc556** [comment]


**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by SchwartzLizer** [comment]
@jcfrog 
I would like to get refund for my Chrome key too. 
Chrome serial number: c7b71b4150 
I bought at 22 Jan 2025 and i will buy firefox version

Thanks

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by TESHEKAS** [comment]
@jcfrog
I would like to get refund for my EDGE key.
Edge serial number: 6c1b14324c
I bought at 19 Feb 2025 
Unfortunately it didn't work well for me

Thanks

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by Djchitown** [comment]
Hi I would like a refund for Chrome also and want to use FireFox.  The first 10 characters of my Chrome license is: fa7394b9d8

Thanks,

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Reply by Djchitown** [reply]
> Hi, You have been refunded. Should be available in a few business days.

All right, thank you for the great product and great customer service. It certainly would be great if the license could be used within all browser types. That way it will prevent you all from having chargebacks or maybe having something pop up in the Chrome store stadium that this works best using Firefox, but I guess Google would watch you to mention that the software doesn't work on their videos

**Comment by TylerGab00** [comment]
@jcfrog  I would like refunded for my chrome key. First 10 digits of the key 7d27b7c1cb. License to cubsfan7919@gmail.com. 
Thank You

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by v0v4iknet** [comment]
Hi! The same problem. Would like to buy a Chrome version*) Key: da5a4fc8b4 Thank you!

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by wlvs** [comment]
I would like to request a refund for my license.
Edge serial number: 6291e8a00c (first 10 digits)
I bought at 27 Feb 2025

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by agustindelamorena** [comment]
I would like refunded for my chrome key. First 10 digits of the key f5bdc74ec7

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.

**Reply by agustindelamorena** [reply]
Hi, I don't know why I received the refund and then another paid was made while I didn't paid another time

**Reply by jcfrog** [reply]
Sorry, I don't see the other transaction you are talking about.
You can send more details at licenses@downloadhelper.net

**Comment by raychak** [comment]
@jcfrog I would like refunded for my chrome key and switch to Firefox. First 10 digits of the key c0b780e2fa. License to [mullard@gmail.com].
Thank You

**Reply by jcfrog** [reply]
I'm sorry, refund is only available the first month.

**Reply by raychak** [reply]
Chrome not work with Youtube anymore... it works on October 2024 when I purchased the license, but not work anymore at end of November 2024 🥲 Any discounts on Firefox extension in this case? You can disable my Chrome license after I bought the Firefox license if needed...

**Reply by jcfrog** [reply]
Sorry but Chrome never worked with Youtube, that's why Chrome premium is cheaper.

**Comment by pmat73** [comment]
Just realized that the Firefox version has more possibilities (fisrt though the licence was navigator agnostic !) ; please refung my Chrome licence 
Here are my fisrt 10 digits licence number : 6122a953ae
Many thanks in advance !

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Reply by pmat73** [reply]
all good, thanks !

**Comment by negreteorg** [comment]


**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by flowexplorations** [comment]
Hello, same situations for me. Just bought the chrome one, loved it, until I found I couldn't use YouTube. Please refund my chrome license starting in: 92e81ae0ed. I just purchased the Firefox one. Thanks for the help!

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Reply by flowexplorations** [reply]
Thank you! Keep up the awesome work

**Reply by felipy361** [reply]
Olá, a situação é a mesma. Acabei de comprar o Chrome e adorei, até descobrir que não consegui usar o YouTube Por favor, devolvam minha licença do Chrome a partir de:16d00b4e1f

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by phamtoanphuc** [comment]
Hello, I want to change my license to Firefox too, pls refund my Chrome license: b714d2c9d5

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by ghost** [comment]
Same. Want license change to Firefox. Chrome license: 95b68ed7f9
Thank you

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by cenkut** [comment]
Same. want license change to firefox. chrome license : bd578d6575
thanks

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by YUDONG7727** [comment]
I would like to get refund for my Chrome key too.
Chrome serial number: d5f4fced5a

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by luizameneghim** [comment]
I would like to be refunded for the Chrome Version in order to buy the Firefox version.
bc4e5ea8d3

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by Samsi92** [comment]
same for me! Thank you for a refund!!

4737dcb53e

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Reply by gukrieger** [reply]
I would like to be refunded for the Chrome Version in order to buy the Firefox version.
50017ee219

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by gukrieger** [comment]
I would like to be refunded for the Chrome Version in order to buy the Firefox version.
50017ee219

**Comment by thebfam** [comment]
Hi, I would like to request a refund for my license, please.
First 10 Numbers
26ae0d7716

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Reply by thebfam** [reply]
Thank you so much!

**Comment by flc1987** [comment]
I bought the Edge license today, but I can't download on YouTube. In Firefox there is a beta version that is possible. I would like to request a refund for my license so I can purchase it in Firefox.

The first 10 digits are: 8a27d1afa0

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by flc1987** [comment]
这是一封自动回复邮件。已经收到您的来信，我会尽快回复。

**Comment by szymonsowinskii** [comment]
Hi, I would like to request a refund for my license for Chrome, please.
First 10 Numbers
8aaac69d03

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by WraithWalker95** [comment]
Hi, I would also like to ask for a refund of my license.
I accidently bought the wrong one.
First 10 digits are: 31826ab1e9

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Reply by WraithWalker95** [reply]
Thank you So Much!! I have Purchased the Chrome one now Thanks!!!

**Comment by protogames-gh** [comment]
Hi, I would also like to ask for a refund of my license.
I accidentally bought the wrong one.
First 10 digits are: 6f54871889

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by Yunianto7905** [comment]
Hi jcfrog,
I want to convert my Chrome license to Firefox license.. my first 10 digit license is : a436458a1b
If it's not possible to directl convert, please refund my Chrome license and I'll buy the Firefox license..
Thank you very much

regards,
Yunianto

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by houssamammari-glitch** [comment]
Hello, i thought i purchased the firefox license but it was the chromr license, can you please refund me so i purchase the one for firefox?
First 10 digits: 90bb96dc09

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by froggygofast** [comment]
Hi, I recently made a similar mistake purchasing a license for Google Chrome, mistakenly assuming that I could use it with Firefox as well. Would it be possible for my purchase to be refunded so I can buy the Firefox version instead?

Chrome License Key: 764a6e1364

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by WannaCryi** [comment]
I make the same mistake, can y just convert it for firefox ?
Chrome: 73e5f230e7

**Reply by jcfrog** [reply]
Sorry we can't swap licenses.
Chrome and Firefox premium are separate products with different prices and features.
What I can do is refund your Chrome premium so that you can purchase Firefox premium if you want to.
Do you want me to do this ?

**Comment by WannaCryi** [comment]
If I can pay the difference with a paypal, otherwise yes we can proceed with a refund


Sent from Yahoo Mail for iPhone


On Monday, August 25, 2025, 10:08, jcfrog ***@***.***> wrote:




Sorry we can't swap licenses.
Chrome and Firefox premium are separate products with different prices and features.
What I can do is refund your Chrome premium so that you can purchase Firefox premium if you want to.
Do you want me to do this ?

**Reply by jcfrog** [reply]
You have been refunded. Should be available in a few business days.

**Comment by WannaCryi** [comment]
Thank you ! And sorry for my mistake ! I think chrome works like Firefox. 


Sent from Yahoo Mail for iPhone


On Monday, August 25, 2025, 10:25, jcfrog ***@***.***> wrote:




You have been refunded. Should be available in a few business days.

**Comment by dptemari-dotcom** [comment]
Hi, I would like to request a refund for my license, please.
First 10 Numbers
4240e55d52

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by eom2012es** [comment]
Hi,
I bought the Chrome version a while ago but the Chrome version stopped working a while ago, but I see that the Edge version works. Is it possible to pay only the difference and not have to pay the whole thing again?

Chrome: cebda7cbfb...

**Reply by jcfrog** [reply]
Sorry it's not possible. What do you mean "stopped working"?

**Reply by eom2012es** [reply]
It doesn´t works with Udemy, neither with YouTube. Not even with other platforms, it keeps giving the same error all the time. We try to support as many websites as possible, so it would help a lot if you could report that error (it's anonymous!). 

I have been waiting to see if this problem would be resolved and nothing. I have tried uninstalling the addon and reinstalling it without results. In summary, it no longer works for me in Chrome.

What solution is there to this persistent problem.

**Reply by jcfrog** [reply]
For udemy please update your CoApp to 2.0.20
For Youtube it is not allowed with Chrome.

**Comment by iskabanov108** [comment]
Hello

I just bought the Firefox license and it didn't work for my Yandex browser.
Then I bought Chrome license and it worked!

Please refund me Firefox license with starting 10 digits: 8bcd92220a

Thank you very much!

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.
